docker/lnd/Dockerfile

```diff
@@ -1,18 +1,15 @@
-FROM golang:1.10-alpine as builder
-
-MAINTAINER Olaoluwa Osuntokun <lightning.engineering>
-
-# Copy in the local repository to build from.
-COPY . /go/src/github.com/lightningnetwork/lnd
```

Comment:
Is there a reason to clone from the repo on L12 (which requires http connection) instead of using COPY?